.env

```diff
@@ -1,6 +1,6 @@
 POSTGRES_USER=user_name
 POSTGRES_PASSWORD=change_password
-POSTGRES_HOST=resources-postgres
+POSTGRES_HOST=resources-psql
```

Comment:
This was broken but I think it went unnoticed because we were using old containers locally. It broke as soon as I did a `make nuke && make fresh-restart`